wiki/Interface/nl.md

```diff
@@ -0,0 +1,304 @@
+
+- Druk op `1` of klik op de `Manage Collections` knop om het Collections scherm te openen - Hier kun je je collecties beheren en de geselecteerde beatmap toevoegen aan of verwijderen van een collectie.
+- Druk op `2` of klik op `Delete...` om de \[1\] geselecteerde mapset te verwijderen, \[2\] de geselecteerde moeilijkheid van een mapset te verwijderen of \[3\] alle beatmaps te verwijderen. De laatste vraagt je vervolgens nog een keer of je zeker weet dat je al je beatmaps wilt verwijderen.
+  - Opmerking: verwijderde beatmaps worden verplaatst naar de prullenbak.
+- Druk op `3` of klik op `Remove from Unplayed` om een ongespeelde beatmap te markeren als gespeeld (dit houdt in dat de kleur van de box veranderd van roze naar oranje).
+- Druk op `4` of klik op `Clear local scores` om alle records van de scores die je behaalt op deze beatmap te verwijderen.
+- Druk op `5` of klik op `Edit` om de geselecteerde beatmap te openen in osu!'s Editor.
+- Druk op `6` of `Esc` of klik op `Close` om terug te keren naar het liedjes-selectie scherm.
+
+Klik op **your user panel** voor toegang to het **User Options Menu**.
+
+Klik op de **[osu!cookie](/wiki/Glossary#cookie)** om **de geselecteerde beatmap te spelen**.
+
+## Ranking Screen
+
+### Grade Screen
+
+![](img/Accuracy_osu!.jpg "Accuracy in osu!")
+
+Dit is het scherm dat je te zien krijgt wanneer je succesvol een map hebt gepast. Je kunt je online resultaten bekijk door naar beneden te scrollen.
+
+**Opmerking:** Het Grade scherm kan veranderen als je een andere skin gebruikt.
+
+Hieronder zijn Grade schermen van de andere gamemodes te zien.
+
+![](img/Accuracy_Taiko.jpg "Accuracy in osu!taiko")
+
+![](img/Accuracy_Mania.jpg "Accuracy in osu!mania")
+
+![](img/Accuracy_CtB.jpg "Accuracy in osu!catch")
+
+### Online Rangschikking
+
+![](img/osu!_online_score_updated2020.jpg "Voorbeeld van een osu!online score")
+
+Dit is jouw online leaderboard. Je kunt hier komen door naar beneden te scrollen op het resultatenscherm. De lokale rangschikking zal jouw naam en score tonen zoals gebruikelijk.
+
+1. Verandeer de naam van het record van de replay hier. Jouw spelersinformatie, je kunt hier jouw [PP](/wiki/Performance_Points), Globale Rang, Totale Score, Overall [Accuracy](/wiki/Accuracy), en level balk zien.
+2. Opties met betrekking tot de beatmap. Je kunt je favoriete beatmaps vinden op je profiel onder Beatmaps->Favourite Beatmaps.
+   1. Export Replay als `.osr`: Slaat de replay op zodat deze online kan worden gebruikt.
+   2. Online Favourite: Voegt de beatmap toe aan jouw favorieten. 
+3. Hier is het verschil te zien wat jouw score verandert aan je profiel. Je kunt hier zien hoeveel je bent geklommen of gedaald in de Globale Rangschikking, hoeveel score is toegevoegd aan je Totale Score en hoeveel je Gemiddelde Accuracy is verandert. De tekst vervaagt erg snel, dus mis het niet!
+4. Spreekt voor zich. Deze balk komt niet in beeld wanneer je offline speelt of de beatmap niet is geupload.
```

Comment:
It looks like you are missing the items under this point, lines 298 - 303 from the English version.